requirements.yml

```diff
@@ -74,8 +74,9 @@
 - src: ome.postgresql
   version: 5.0.1
 
-- src: ome.postgresql_backup
-  version: 0.1.1
+- name: ome.postgresql_backup
```

Comment:
This can be updated to `ome.postgresql_backup 0.2.0` which has been released with compression support.